packages/nimble-blazor/CONTRIBUTING.md

```diff
@@ -75,10 +75,20 @@ The C# code for a property supporting 2-way binding will look like this:
 
 ## Testing
 
-### Automated
+### Automated Unit Tests (NimbleBlazor.Tests)
```

Comment:
nit: in all of these headers avoid the extra punctuation, makes for poorer quality links. can just move it into the paragraph

```suggestion
### Automated Unit Tests

Namespace:  NimbleBlazor.Tests

```

Reply:
ex the current headers anchor link getting really verbose and a bit confusing to read:
```
https://github.com/ni/nimble/blob/<omitted>/packages/nimble-blazor/CONTRIBUTING.md#automated-acceptance-tests-using-playwright-nimbleblazortestsacceptance
```

Reply:
Updated